src/main/java/org/apache/commons/net/nntp/NNTPClient.java

```diff
@@ -583,12 +583,12 @@ private void parseArticlePointer(final String reply, final ArticleInfo pointer)
      * due to malformed headers.
      * <p>
      * You must not issue any commands to the NNTP server (i.e., call any (other methods) until you finish writing to the returned Writer instance and close it.
-     * The NNTP protocol uses the same stream for issuing commands as it does for returning results. Therefore the returned Writer actually writes directly to
+     * The NNTP protocol uses the same stream for issuing commands as it does for returning results. Therefore, the returned Writer actually writes directly to
      * the NNTP connection. After you close the writer, you can execute new commands. If you do not follow these requirements your program will not work
      * properly.
      * <p>
      * Different NNTP servers will require different header formats, but you can use the provided {@link org.apache.commons.net.nntp.SimpleNNTPHeader} class to
-     * construct the bare minimum acceptable header for most news readers. To construct more complicated headers you should refer to RFC 822. When the Java Mail
+     * construct the bare minimum acceptable header for most newsreaders. To construct more complicated headers you should refer to RFC 822. When the Java Mail
```

Comment:
"newsreaders" is not a word AFAIK

Reply:
What about this: https://www.newsgroupreviews.com/newsgroup-readers.html, see title? Saw it more recently in this form.

Reply:
See also: https://en.wikipedia.org/wiki/List_of_Usenet_newsreaders

Reply:
Gross, but ok I guess.